ytfzf

```diff
@@ -537,17 +539,27 @@ get_yt_json () {
 }
 
 get_yt_html () {
-    link="$1"
-    query="$2"
-    printf "%s" "$(
-	curl "$link" -s \
-	  -G --data-urlencode "search_query=$query" \
-	  -G --data-urlencode "sp=$sp" \
-	  -H 'authority: www.youtube.com' \
-	  -H "user-agent: $useragent" \
-	  -H 'accept-language: en-US,en;q=0.9' \
-	  --compressed
-    )"
+    local link="$1"
+    local query="$2"
+	if [ "$use_wget" -eq 1 ]; then
+		printf "%s" "$(
+		wget -O - --no-cookie --quiet \
+		  --header 'authority: www.youtube.com' \
+		  --header "user-agent: $useragent" \
+		  --header 'accept-language: en-US,en;q=0.9' \
+		  "${link}?search_query=${query}&sp=${sp}" \
+		)"
```

Comment:
why printf the stdout?
We could omit printf since wget will output to stdout.
Similarly with the curl below